<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>docs_card</class>
 <widget class="QDialog" name="docs_card">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>570</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Document</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>â„–</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_num">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Date</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dateEdit_doc">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Title of document</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_name"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Organization</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_org"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QToolButton" name="toolButton_col">
         <property name="text">
          <string>...</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::LeftArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_add_book">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_column">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="checkBox_isbn">
              <property name="text">
               <string>ISBN</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_title">
              <property name="text">
               <string>Title</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_auth">
              <property name="text">
               <string>Author</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_pub">
              <property name="text">
               <string>Publisher</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_year">
              <property name="text">
               <string>Year</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_count">
              <property name="text">
               <string>Identifier</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_book">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>23</number>
         </attribute>
         <column>
          <property name="text">
           <string>id_card</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>id_book</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ISBN</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Title</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Author</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Publisher</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Year</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Identifier</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Note:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_note"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Total:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_total">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="pushButton_del">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Del</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_toFirst">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_toPrev">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_toNext">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_toLast">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_save">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_close">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_status">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
